src/frontend/package.json

```diff
@@ -43,7 +43,7 @@
     "@typescript-eslint/eslint-plugin": "5.40.0",
     "@typescript-eslint/parser": "5.40.0",
     "@vitejs/plugin-react": "2.1.0",
-    "cypress": "10.10.0",
+    "cypress": "^12.10.0",
```

Comment:
that do not use **^** sign here, just use 12.10.0.

Reply:
Hello @binggaofw - Corrected the same! Thanks!
    "cypress": "^12.10.0",
    "cypress": "12.10.0",